src/swarm-scenario/Swarm/Game/Device.hs

```diff
@@ -71,7 +71,7 @@ data CapabilityCost e = CapabilityCost
 -- Otherwise, parse as a Map from capabilities to ingredients.
 instance (FromJSON e) => FromJSON (SingleEntityCapabilities e) where
   parseJSON x =
-    simpleList <|> (Capabilities <$> costMap)
```

Comment:
@kostmo I swapped these to get better errors, as they only told me that the wrong capability is not an object.

I don't know if this can be improved much. Maybe the costly capabilities could be under a separate key.